keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1918,6 +1932,13 @@ currently declared on the Pod and Device Plugin level.
 Extending the Device Plugins API to use [Container Device Interface](https://github.com/container-orchestrated-devices/container-device-interface)
 would help address some of the requirements, but not all of them.
 
+NodePrepareResource and NodeUnprepareResource could be added as optional
+calls to the Device Plugins API. However, this would mean that
```

Comment:
Let's be clearer about what "optional calls" means:

```suggestion
NodePrepareResource and NodeUnprepareResource could be added to the Device Plugins API and only get called for
resource claims.

However, this would mean that
```